### ISSUE TITLE
This library is not being maintained

### ISSUE THREAD

**Issue by Johnrobmiller** [eduardolundgren/tracking.js#409]
I'm seeing issues like this (https://github.com/eduardolundgren/tracking.js/issues/388) that are four years old and are obviously closable as soon as you look at it. I'm also seeing 24 pull reqeusts, all but one dating back to 2020. This is a huge red flag.

I think it's safe to say that this library has not been maintained for at least four years and should probably be avoided. Even if someone started maintaining it today, there's so much backlog now that it would still be a long time before any reasonable person would trust using it.

**Comment by eduardolundgren** [comment]
You're right; the project hasn't seen much activity for about five years. We've been on the lookout for contributors during this time, but unfortunately, we haven't had much luck finding anyone to help carry the torch.

If you're interested or know someone who might be, we'd love to get help.

Thanks for your understanding and for reaching out.
